src/test/scala/firrtlTests/ClockListTests.scala

```diff
@@ -82,6 +82,37 @@ class ClockListTests extends FirrtlFlatSpec {
     }
     val writer = new StringWriter()
     val retC = new ClockList("HTop", writer).run(c)
-    (writer.toString()) should be (check)
+    (writer.toString) should be (check)
```

Comment:
This is no big deal, but the parens around writer.toString are flagged as unnecessary